.github/workflows/prannouncer.yml

```diff
@@ -0,0 +1,14 @@
+name: "Google Chats PR Announcer"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 6 * * 1-5"
+
+jobs:
+  stale:
```

Comment:
why call it `stale`?

Reply:
Woops, good catch, I was lazy and copied the action from the stale PR action I previously made!